spectator/sidecarwriter.py

```diff
@@ -119,9 +120,13 @@ class UdpWriter(SidecarWriter):
     def __init__(self, location: str, address: Tuple[str, int]) -> None:
         super().__init__(location)
         self._address = address
+        if type(ip_address(self._address[0])) is IPv4Address:
```

Comment:
Not sure how much of a concern it is, but this would force the address to be an IP. Are there any use-cases where we might need to use a hostname?

Reply:
Ideally it could be made transparent for the user. I think the current approach in this PR is fine for now and maybe we'll come up with a better solution going forward.